modules/aleph/src/Aleph/AlephMaterial.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace Drupal\aleph\Aleph;
+
+use DateTime;
+
+/**
+ * Class AlephMaterial.
+ *
+ * @package Drupal\aleph\Aleph
+ */
+class AlephMaterial {
+
+  private $loans;
```

Comment:
Set them as protected, that way factories in the same namespace and sub-classes can mutate them. As a rule of thumb, avoid using private unless you are 100% terrible things will happen if anyone else touches the property.